src/tests/ftest/datamover/dm_posix_meta_entry.py

```diff
@@ -76,7 +76,7 @@ def run_dm_posix_meta_entry(self, tool):
             self.dfuse.mount_dir.value, pool1.uuid, posix_cont.uuid)
 
         # Create 1 source posix path with test data
-        posix_src_path = self.new_posix_test_path(False, posix_root)
+        posix_src_path = self.new_posix_test_path(parent=self.workdir)
```

Comment:
I think we should be able to remove posix_root and posix_cont above now, since we are using self.workdir instead of posix_root